README.md

```diff
@@ -44,7 +44,8 @@ including
 
 
 # Document
-- [6.x Documents](docs/README.md).
+- [6.x Documents](https://github.com/apache/skywalking/blob/6.x/docs/README.md).
```

Comment:
Move this as the second one. 

The first line of this page includes 6 too, https://github.com/apache/skywalking/blob/master/docs/README.md

Reply:
updated